man/man7/zfsprops.7

```diff
@@ -1495,6 +1495,16 @@ of user data can be lost if a single on-disk block is corrupt.
 The exact behavior of which metadata blocks are stored redundantly may change in
 future releases.
 .Pp
+When set to
+.Sy some ,
+ZFS stores an extra copy of few types of critical metadata.
+If a single on-disk block is corrupt, at worst a single user file can be lost.
```

Comment:
I'd add, "This can improve file create performance since less metadata needs to be written."

Reply:
done.